mw-webapp/src/pages/allWaysPage/AllWaysPage.tsx

```diff
@@ -0,0 +1,49 @@
+import {useEffect, useState} from "react";
+import {WayDTO} from "src/model/DTOModel/WayDTO";
+import {ReportsTable} from "src/pages/reportsTable/ReportsTable";
+import {WayService} from "src/service/WayService";
+
+/**
+ * Page with all Ways
+ */
+export const AllWaysPage = () => {
+  const [ways, setWays] = useState<WayDTO[] | null>(null);
+
+  /**
+   * Get all ways
+   */
+  const getAllWays = async () => {
+    const waysDTO = await WayService.getWaysDTO();
+    setWays(waysDTO);
+  };
+
+  useEffect(() => {
+    getAllWays();
+  }, []);
+
+  return (
+    <div>
+      {ways
+        ? (
+          ways.map((way) => (
+            <>
```

Comment:
I think it's probably better to use Fragment instead of a <></> and already use the key in it since it's the parent element

Reply:
Done